tests/test_load_score.py

```diff
@@ -37,9 +37,9 @@ def test_load_score(self):
             + MATCH_IMPORT_EXPORT_TESTFILES
             + EXAMPLE_FILES
         ):
-            self.load_score(fn)
+            self.check_return_type(fn)
 
-    def load_score(self, fn):
+    def check_return_type(self, fn):
         try:
```

Comment:
A try-except is not a nice test. I would prefer if we let it fail from the start, but this is not related to this PR.

Reply:
I still corrected it to a better format